src/basic/path-util.c

```diff
@@ -1274,7 +1274,8 @@ bool is_device_path(const char *path) {
         /* Returns true for paths that likely refer to a device, either by path in sysfs or to something in
          * /dev. */
```

Comment:
Please extend the comment that this refuses `/dev`.

Reply:
I extended the comment and added an example.